src/policy/semantic.rs

```diff
@@ -401,16 +401,16 @@ impl<Pk: MiniscriptKey> Policy<Pk> {
     }
 
     /// Helper function to detect a true/trivial policy
-    pub fn is_trivial(&self) -> bool {
-        match *self {
+    pub fn is_trivial(self) -> bool {
+        match self.normalized() {
```

Comment:
This function is now very expensive and may fail for large policies

Reply:
My rationale was trivial should return `true` for the normalized policy. Or(true, largepolicy) should also be trivial